app/assets/javascripts/discourse/app/components/bookmark-menu.gjs

```diff
@@ -199,18 +212,31 @@ export default class BookmarkMenu extends Component {
       </:trigger>
       <:content>
         <div class="bookmark-menu__body">
+
+          {{#unless this.showEditDeleteMenu}}
+            <div class="bookmark-menu__title">{{icon "check-circle"}}<span
+              >{{i18n "bookmarks.bookmarked_success"}}</span>
+            </div>
+          {{/unless}}
+
           {{#if this.showEditDeleteMenu}}
+            {{#if this.site.mobileView}}
+              <div class="bookmark-menu__title">{{icon "bookmark"}}<span>{{i18n
+                    "bookmarks.bookmark"
+                  }}</span>
+              </div>
+            {{/if}}
             <ul class="bookmark-menu__actions">
               <li class="bookmark-menu__row -edit" data-menu-option-id="edit">
                 <DButton
                   @icon="pencil-alt"
                   @label="edit"
                   @action={{this.onEditBookmark}}
-                  @class="bookmark-menu__row-btn btn-flat"
+                  @class="bookmark-menu__row-btn btn-transparent"
                 />
               </li>
               <li
-                class="bookmark-menu__row -remove"
+                class="bookmark-menu__row --remove"
```

Comment:
OK I know there was some discussion on this, so did we end up deciding to use `--` for modifiers in CSS instead of just `-`? I don't mind either way, I just would like to know for future reference.

Reply:
I will let @chapoi answer